generative_models/VanillaGAN_TensorFlow/models.py

```diff
@@ -0,0 +1,59 @@
+import tensorflow as tf
+from tensorflow.keras.layers import Dense, Dropout, Input, BatchNormalization,LeakyReLU
+from tensorflow.keras.models import Model,Sequential
+from tensorflow.keras.optimizers import Adam
+
+def adam_optimizer():
+    return Adam(lr=0.0002, beta_1=0.5)
+
+def create_generator():
+
+    generator=Sequential()
+
+    generator.add(Dense(units=256,input_dim=100))
```

Comment:
add encoding_dims as parser args and change according to arg instead of hardcoding it to 100

Reply:
done